C7Engine/AI/UnitAI/CombatAI.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections.Generic;
+using C7GameData;
+using C7GameData.AIData;
+using Serilog;
+
+namespace C7Engine.AI.UnitAI {
+	/// <summary>
+	/// A unit whose intended role is combat, other than purely point-defense combat.
+	///
+	/// This is likely to evolve significantly, and likely have sub-classes; this is the very first iteration.
+	/// This first iteration is focused on defeating barbarians.
+	/// </summary>
+	public class CombatAI : C7Engine.UnitAI {
+		private Tile target;
+
+		private ILogger log = Log.ForContext<DefenderAI>();
```

Comment:
wrong context again

Reply:
Should be fixed.  Really wish there were a way to make that automatic.